astroquery/noirlab/__init__.py

```diff
@@ -0,0 +1,87 @@
+from astropy import config as _config
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""NSF's OIR Lab Astro Data Archive (Beta)
+------------------------------------------
+
+The NSF's OIR Lab Astro Data Archive (formerly NOAO Science Archive)
+provides access to data taken with more than 40 telescope and
+instrument combinations, including those operated in partnership with
+the WIYN, SOAR and SMARTS consortia, from semester 2004B to the
+present. In addition to raw data, pipeline-reduced data products from
+the DECam, Mosaic and NEWFIRM imagers are also available, as well as
+advanced data products delivered by teams carrying out surveys and
+other large observing programs with NSF OIR Lab facilities.
+
+Total holdings 9,634,110  V.3
```

Comment:
is there a way to auto generate this table? I also feel that being hidden in the this docstring (that is exposed nowhere) is not the ideal place for it, and it should be moved to the narrative docs.

Keeping the Acknowledgement here on the other hand is preferred. Or in fact, you can even repeat it in a section in the narrative docs, too.